### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by benmenachem**
Updated README.md file to include project and team introduction.

### CODE REVIEW

**Review by guyy13 — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+
+
+# Student JMaster 
+
+Student jobs app
+
+# Description
+A platform for students to find jobs/internships.
+The website allows a student to find a job or an internship that is relevant to their major.
+The jobs can be organized according to a few filters, such as experince needed and location.
```
Comment (guy9050) [inline README.md]: experience

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+
+
+# Architecture
+
+| Frontend      | Backennd      | Datebase      |
```
Comment (dmaizel) [inline README.md]: Typos

File: README.md (hunk 1 above)
Comment (benmenachem) [reply README.md]: work experience required for the job if needed

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
...
+| ------------- | ------------- | ------------- |
+| HTML          | Python        | SQLite        |
+| CSS           | Django        |  
+| JavaScript    | Vagrant       |
+| Bootstarp     |
```
Comment (dmaizel) [inline README.md]: Bootstrap*

File: README.md (hunk 1 above)
Comment (guy9050) [reply README.md]: yes I meant you had a typo in that word

Comment (benmenachem) [reply README.md]: My bad 
Will fix it now

**Review by guyy13 — APPROVED** [review]

**Review by eldaraharonson — APPROVED** [review]
Looks very good!

**Comment by guy9050** [comment]
please squash the commits

**Pull request merged**